stuff/redis-dies-test.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+// kill redis-server two times and see if
+// caron stays happy :)
+
+const { spawn } = require('child_process')
+const assert = require('assert')
+
+const Caron = require('../')
+
+const restarts = 2
+let killCount = 0
+
+let victims = []
+function spawnVictim () {
+  // a brand new redis will kill caron
+  const opts = killCount !== restarts ? [] : [] // [ '--conf ./tmp.conf' ]
```

Comment:
a completely fresh redis instance will not work, as caron sends the lua in the constructor and the fresh redis instance misses the rdb dump